io.jamulus.Jamulus.yaml

```diff
@@ -1,6 +1,6 @@
-app-id: io.jamulus.Jamulus
+id: io.jamulus.Jamulus
```

Comment:
Could you reference the docs to explain why this change is necessary?

Reply:
See #81

Reply:
Some flatpak Devs says app prefix was deprecated. But no one send feedback about this in matrix Room

Reply:
Ping @gasevin @TingPing

Reply:
Seems good enough 👍  Have you tested if everything works with this new build?

Reply:
Works